javascript/ql/src/semmle/javascript/Promises.qll

```diff
@@ -659,3 +659,37 @@ private module DynamicImportSteps {
     }
   }
 }
+
+/**
+ * Provides classes modeling libraries implementing `promisify` functions.
+ * That is, functions that convert callback style functions to functions that return a promise.
+ */
+module Promisify {
+  /**
+   * Gets a call to a `promisifyAll` function.
+   * E.g. `require("bluebird").promisifyAll(...)`.
+   */
+  class PromisifyAllCall extends DataFlow::CallNode {
+    PromisifyAllCall() {
+      this =
+        [DataFlow::moduleMember("bluebird", "promisifyAll"), DataFlow::moduleImport("pify")]
```

Comment:
DId you intentionally remove `util-promisifyall` in this PR?

Reply:
No it wasn't.  
I looked through the code one last time before marking the PR as ready for review, and I noticed that [`util-promisifyall`](https://www.npmjs.com/package/util-promisifyall) is not a popular library. 
I thought that I had added it and just forgot about the change-note.  

I'll add it back.